### ISSUE TITLE
(Low priority) Ability to set color for search items.

### ISSUE THREAD

**Issue by jordwalke** [t9md/atom-vim-mode-plus#183]
I'm not sure how search highlighting is implemented, but it's not currently possible to set the `color` css attribute. It appears this will be difficult to implement because the search highlighting in vmp, is simply an overlay of the text buffer. I am filing this github issue just to track the feature request. I expect it wouldn't be easy to do for quite some time.

**Comment by t9md** [comment]
Easy. For example, set following in your `style.less`. you can open with `open your stylesheet` command from command-palette.

```
atom-text-editor::shadow {
  .vim-mode-plus-highlight-search .region {
    border: none;
    background-color: red;
  }
}
``

**Comment by t9md** [comment]
Btw, this kind of TIPS should be consolidated in Wiki so that other user can find without searching whole past issue..

**Comment by jordwalke** [comment]
The `color` css attribute modifies the text color, not background color. I am asking about the css `color` attribute for text.

**Comment by t9md** [comment]
ok, thats not possible as you described.
Pigments package doing very heavy workaround to overlay custom HTML element to achieve changing fore ground color.
Its too much for vmp and it have perf impact greatly.

**Comment by jordwalke** [comment]
I agree - which is why I suggested it would be very difficult, and low priority. `Atom` should make it easier for vmp to highlight regions without needing an "overlay" or "underlay".

**Comment by t9md** [comment]
Interesting enhancement in atom/text-buffer#149

Especially for following part.
> We currently only support a single text decoration layer via DisplayLayer.prototype.setTextDecorationLayer. This will be used in Atom to perform syntax highlighting. However, in the future it should be straightforward to allow multiple text decoration layers, which could simplify the implementation of packages that need to style runs of text in specific ways instead of just highlighting them with absolutely positioned divs. An example of such a package is pigments by @abe33.

When display-layer is introduced to atom( maybe in atom 1.8.0?) and then TextDecorationLayers have support multiple decoration layer in following release, then I think I could easily solve this issue.

Not so far, Atom's advancement is very fast.!
